scripts/linters/general_purpose_linter.py

```diff
@@ -44,6 +44,7 @@
     'core/templates/combined-tests.spec.ts',
     'core/templates/css/oppia-material.css',
     'core/templates/google-analytics.initializer.ts',
+    'extensions/classifiers/proto/*',
```

Comment:
Why this change?

Reply:
To skip the lint checks of auto generated protobuf files. They don't comply with our lint rules.